Kloudy/Kloudy/View/CheckLocation/LocationSelectionCollectionViewCell/LocationSelectionCollectionViewCell.swift

```diff
@@ -58,22 +61,22 @@ class LocationSelectionCollectionViewCell: UICollectionViewCell {
     private func setLayOut() {
         locationNameLabel.snp.makeConstraints {
             $0.leading.equalTo(20)
-            $0.top.bottom.equalTo(36)
+            $0.centerY.equalToSuperview()
         }
         
         weatherImage.snp.makeConstraints {
             $0.leading.equalTo(152)
-            $0.top.bottom.equalTo(35)
+            $0.centerY.equalToSuperview()
         }
         
         temperatureLabel.snp.makeConstraints {
-            $0.trailing.equalTo(20)
-            $0.top.equalTo(40)
+            $0.trailing.equalTo(-20)
```

Comment:
```suggestion
            $0.trailing.equalToSuperview().inset(20)
```

이렇게 바꿔주시는게 snapKit을 조금 더 잘 사용하는 방법 같습니다!

Reply:
감사합니다!